### PR TITLE
[Azure Stack][Admin] Update for Azure Stack 1.5.0

### PR DESCRIPTION
**Pull request by deathly809**
## Description

New Version of Update module.

## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by maddieclayton** [comment]
@bganapa Will approve and merge all 12 of these Stack Admin PRs once they are passing CI and you have signed off on each.

File: src/StackAdmin/Azs.Update.Admin/Module/Azs.Update.Admin/Azs.Update.Admin.psd1 [hunk 1]
```diff
@@ -102,19 +102,32 @@ Licensed under the MIT License. See License.txt in the project root for license
...
             # A URL to the main website for this project.
-            # ProjectUri = ''
+            ProjectUri   = 'https://github.com/Azure/azure-powershell'
 
             # A URL to an icon representing this module.
             # IconUri = ''
 
             # ReleaseNotes of this module
-            # ReleaseNotes = ''
+            ReleaseNotes = '## 2018.8.12 - Version 0.2.0
```
Comment (deathly809) [inline src/StackAdmin/Azs.Update.Admin/Module/Azs.Update.Admin/Azs.Update.Admin.psd1]: Update the date.

Comment (deathly809) [reply src/StackAdmin/Azs.Update.Admin/Module/Azs.Update.Admin/Azs.Update.Admin.psd1]: Will update all dates when we do PR to update AzureStack rollup module.

File: src/StackAdmin/Azs.Update.Admin/Tests/src/UpdateAdmin.Tests.ps1 [hunk 2]
```diff
@@ -165,20 +165,26 @@ InModuleScope Azs.Update.Admin {
...
+
         It "TestGetUpdateLocation" -Skip:$('TestGetUpdateLocation' -in $global:SkippedTests) {
             $global:TestName = "TestGetUpdateLocation"
 
-            $list = Get-AzsUpdateLocation -ResourceGroup System.Redmond -Location redmond
+            $list = Get-AzsUpdateLocation -ResourceGroup $global:ResourceGroupName
+            $list | Should not be $null
             foreach ($location in $list) {
-                $location1 = Get-AzsUpdateLocation -Location $location.Name -ResourceGroup System.Redmond
+                Write-Host $location.Id
```
[the remark this answers is not in the document]
Comment (deathly809) [reply src/StackAdmin/Azs.Update.Admin/Tests/src/UpdateAdmin.Tests.ps1]: Done.

File: src/StackAdmin/Azs.Update.Admin/Tests/src/UpdateAdmin.Tests.ps1 [hunk 3]
```diff
@@ -210,12 +216,14 @@ InModuleScope Azs.Update.Admin {
         it "TestGetUpdateRun" -Skip:$('TestGetUpdateRun' -in $global:SkippedTests) {
             $global:TestName = "TestGetUpdateRun"
 
-            $list = Get-AzsUpdate -ResourceGroup System.Redmond -Location redmond
+            $list = Get-AzsUpdate -ResourceGroup $global:ResourceGroupName -Location $global:Location
+            $list | Should not be $null
             foreach ($update in $list) {
-                $update | fl
-                $runList = Get-AzsUpdateRun -UpdateName $update.Name -ResourceGroup System.Redmond -Location redmond
+                Write-Host "Name: $($update.Name)"
```
[the remark this answers is not in the document]
Comment (deathly809) [reply src/StackAdmin/Azs.Update.Admin/Tests/src/UpdateAdmin.Tests.ps1]: Done.

File: src/StackAdmin/Azs.Update.Admin/Tests/src/UpdateAdmin.Tests.ps1 [hunk 4]
```diff
@@ -210,12 +216,14 @@ InModuleScope Azs.Update.Admin {
...
+                $runList = Get-AzsUpdateRun -UpdateName $update.Name -ResourceGroup $global:ResourceGroupName -Location $global:Location
                 foreach ($run in $runList) {
-                    $run1 = Get-AzsUpdateRun -Name $run.Name -ResourceGroup System.Redmond -Location redmond -UpdateName $update.Name
+                    Write-Host $run.Name
```
[the remark this answers is not in the document]
Comment (deathly809) [reply src/StackAdmin/Azs.Update.Admin/Tests/src/UpdateAdmin.Tests.ps1]: Done.

File: src/StackAdmin/Azs.Update.Admin/Module/Azs.Update.Admin/Azs.Update.Admin.psd1 [hunk 5]
```diff
@@ -102,19 +102,32 @@ Licensed under the MIT License. See License.txt in the project root for license
...
+            * Module dependencies updated
+                * AzureRM.Profile >= 5.3.4
```
Comment (bganapa) [inline src/StackAdmin/Azs.Update.Admin/Module/Azs.Update.Admin/Azs.Update.Admin.psd1]: Profile versions

File: src/StackAdmin/Azs.Update.Admin/Module/Azs.Update.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Resume-AzsUpdateRun.ps1 [hunk 6]
```diff
@@ -97,6 +95,8 @@ function Resume-AzsUpdateRun {
             $Location = $ArmResourceIdParameterValues['updateLocation']
             $UpdateName = $ArmResourceIdParameterValues['update']
             $Name = $ArmResourceIdParameterValues['runId']
+        } else {
+            $Name = Get-ResourceNameSuffix -ResourceName $Name
```
Comment (bganapa) [inline src/StackAdmin/Azs.Update.Admin/Module/Azs.Update.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Resume-AzsUpdateRun.ps1]: why is this needed?

File: src/StackAdmin/Azs.Update.Admin/Tests/packages.config [hunk 7]
```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.AzureStack.Management.Update.Admin" version="0.1.0-preview" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime.Azure.TestFramework" version="1.6.0" targetFramework="net45" />
-</packages>
\ No newline at end of file
+  <package id="Microsoft.Rest.ClientRuntime.Azure.TestFramework" version="1.7.2" targetFramework="net452" />
```
Comment (bganapa) [inline src/StackAdmin/Azs.Update.Admin/Tests/packages.config]: lets take dependency on the common targets
